docs/source/user/extensions.md

```diff
@@ -70,26 +88,26 @@ jupyter labextension enable <foo>
 Core plugins can also be disabled (and then re-enabled).
 
 
-## Advanced usage
+## Advanced Usage
```

Comment:
Do we want to subdivide this? the the title is not necessarily descriptive of what kinds of advanced settings a user will learn about

Reply:
Okay, I have restructured and subdivided a bunch of sections.